lib/tx.js

```diff
@@ -82,7 +82,7 @@ function customerHistory (customerId, thresholdDays) {
       FROM cash_out_txs txOut
       WHERE txOut.customer_id = $1
       AND txOut.created > now() - interval $2
-      AND (error_code IS NULL OR error_code NOT IN ('operatorCancel', 'scoreThresholdReached', 'ciphertraceError'))
+      AND (error_code IS NULL OR error_code NOT IN ('operatorCancel', 'scoreThresholdReached', 'walletScoringError', 'ciphertraceError'))
```

Comment:
If Ciphertrace went down the drain maybe create a migration to `ciphertraceError -> walletScoringError` ?

Reply:
Good point, done.